rfcs/0017-long-fork-consensus.md

```diff
@@ -0,0 +1,123 @@
+## Summary
+[summary]: #summary
+
+We currently do not support long forks. This means that to join the network you need a recent checkpoint, in other words to have always been online or to trust another party that has always been online.
+
+This adds support for a client to join the network at any point of time, regardless of if its been online recently. It adds this safely under a [50% + epsilon] security assumption. It also includes a mechanism to restore a chain to strength if this assumption was to be temporarily violated.
+
+This mechanism also does not require checkpointing at any time horizon, compared to alternatives.
```

Comment:
Do you mean `within a time horizon`? As in, this mechanism is free from the previous restriction that we a node would need to be aware of some trusted checkpoint within a certain amount of time (previously ~1 year) of the state is is selecting?

Reply:
That is correct, no trusted checkpoint necessary :)

Reply:
I'm suggesting an update to the language for clarity.